webpack.config.js

```diff
@@ -65,9 +91,33 @@ const config = {
                     ]
                 }
             },
+            {
+                test: /\.css$/,
+                use: ["style-loader", "css-loader"],
+            },
+            // Needed for loading CSS files with url()s
+            {
+                test: /\.(jpe?g|png|gif)$/i,
+                loader:"file-loader",
+                options:{
+                  name:'[name].[ext]',
+                  outputPath:'../../build/images/css/'
+                }
+            },
+            // Needed for loading the Bootstrap font file with WOFF/SVGs
+            {
+                test: /\.(woff|woff2|eot|ttf|svg)$/,
+                loader: 'url-loader?limit=100000'
+            },
         ],
     },
     plugins: [
```

Comment:
Usually people extract the CSS back out of the bundles using either ExtractTextPlugin or MiniCssPlugin.  I'm cool with not doing that.  I'm not sure how keeping the CSS in the bundles affects our ability to override it in webapp.  Maybe that's necessary give that we're using different templates in webapp.